federation-2/harmonizer/deno/do_compose.js

```diff
@@ -56,7 +56,10 @@ try {
   let hints = [];
   if (composed.hints) {
     composed.hints.map((composed_hint) => {
-      hints.push({ message: composed_hint.toString() });
+      hints.push({
+        message: composed_hint.toString(),
+        level: composed_hint.definition.level,
```

Comment:
it looks like we are putting that info right into the JSON object here in the JS code - can we just do something like 

```rust
let data = serde_json::from_string(data_from_js);
let message = data["message"];
let level = data["level"];
```

?